_includes/navbar.html

```diff
@@ -0,0 +1,2 @@
+<div class="nav-sidebar">
+</div>
```

Comment:
I think this should be located in `_includes/sidenav.html`, as the purpose of this include is for the main top navigation.